examples/hooks/RecordHooks/conditionallyEmptyField.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { NumberField, TextField, Sheet, Workbook } from '@flatfile/configure'
+import { SheetTester } from '../../../src/utils/testing/SheetTester'
+
+//this is a basic example of how to modify a field based on other fields
+const testSheet = new Sheet(
+  'Test Sheet',
+  {
+    paymentStatus: TextField(),
+    amount: NumberField(),
+  },
+  {
+    //because this hook is comparing multiple fields and both modifying a field and adding a message, it must be run as a recordHook rather than a fieldHook
```

Comment:
I would recommend:
//because this hook is comparing multiple fields and both modifying a field, it must be run as a recordHook rather than a fieldHook

Adding a message at the same time shouldn't be promoted as a reason to use `recordCompute`...   The only way you could properly set the warning message for 'payment' status is by looking at 'paymentStatus' and 'amount', that's why you want recordCompute.